### DISCUSSION TITLE
Font Size

### DISCUSSION THREAD

**Discussion by chazbz** [ImageMagick/ImageMagick#5636] [General]
I am running Linux Mint 20.3 and I have ImageMagick and I would use the program more often I could see the the tiny font on the buttons without using a magnifying glass. Is there away to enlarge the font of the buttons and the associated instructions that come with clicking the buttons?

**Comment by fmw42** [comment]
Which tool are you using?  Please Showa screen snap or provide the link to what you are using. Mostly Imagemagick is command line processing run from the terminal.  On some Linux you can open a GUI to do some processing?  There is also a web interface?  Which one are you using?

**Reply by fmw42** [reply]
Have you tried changing your system default font? Or the Imagemagick font. See last line at https://imagemagick.org/source/policy.xml regarding setting system font.

**Comment by chazbz** [comment]
![Screenshot from 2022-10-08 16-02-25](https://user-images.githubusercontent.com/111151463/194727784-fa9e3a29-eaa6-4d56-ba55-970edda1fae3.png)

**Reply by fmw42** [reply]
That is the GUI.  If changing the font in the policy.xml file does not help, then one of the IM developers will need to answer. But that font is not so small in my opinion.

**Reply by fmw42** [reply]
You can change the font size in your internet browser, can you not?  Most browsers let you zoom in or out.  Or change your display resolution.

**Reply by chazbz** [reply]
fmw42, you are probably not 75 years old either

**Reply by fmw42** [reply]
77 But I wear glasses when at my computer.

**Reply by fmw42** [reply]
Change your monitor display resolution.  What resolution are you using?

**Reply by chazbz** [reply]
Monitor Display resolution would change everything else on my computer. I think I'll just let it go for now and use a magnifying glass.

**Reply by fmw42** [reply]
One of the IM developers will need to respond on this question.
